src/MusicStore/test/MusicStore.E2ETests/SmokeTests.cs

```diff
@@ -22,18 +22,17 @@ public static TestMatrix TestVariants
                 .WithAllAncmVersions()
                 .WithAllHostingModels();
 
-        // ANCM In-process cannot run on netcoreapp2.1 and below
+        // In-proc is .NET Core only
         public static TestMatrix TestVariantsWithoutInproc
             => TestMatrix.ForServers(ServerType.IISExpress, ServerType.Kestrel, ServerType.HttpSys)
-                .WithTfms(Tfm.NetCoreApp21, Tfm.NetCoreApp20, Tfm.Net461)
+                .WithTfms(Tfm.Net461)
```

Comment:
I think this can be consolidated into a single TestVariant. I'm pretty sure the TestMatrix knows that inproc + net461 doesn't work.

Reply:
Cool beans. Updated.